_ml-commons-plugin/api/model-apis/batch-predict.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+title:  Batch predict
+parent: Model APIs
+grand_parent: ML Commons APIs
+nav_order: 65
+---
+
+# Batch predict
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/ml-commons/issues/2488).
+{: .warning}
+
+ML Commons can perform inference on large datasets in an offline asynchronous mode using a model deployed on external model servers. To use the Batch Predict API, you must provide the `model_id` for an externally hosted model. Currently, only Amazon SageMaker, Cohere, and OpenAI are the verified external servers that support this API.
+
+For information about user access for this API, see [Model access control considerations]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/index/#model-access-control-considerations).
+
+For information about externally hosted models, see [Connecting to externally hosted models]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/index/). 
+
```

Comment:
Does the information below need to be in the form of a bulleted list, or could it just be a sentence?

Reply:
Not necessarily but if we add more connectors, I think it's better if it's a list.